packages/skuba-dive/.dockerignore

```diff
@@ -0,0 +1,23 @@
+# managed by skuba
+.gantry/
+.git/
+.idea/
+.serverless/
+.vscode/
+node_modules*/
+
+/coverage*/
+/dist*/
+/lib*/
+/tmp*/
+
+.DS_Store
+npm-debug.log
+yarn-error.log
+# end managed by skuba
+
+# Ignore .npmrc. This is no longer managed by skuba as pnpm projects use a managed .npmrc.
+# IMPORTANT: if migrating to pnpm, remove this line and add an .npmrc IN THE SAME COMMIT.
+# You can use `skuba format` to generate the file or otherwise commit an empty file.
+# Doing so will conflict with a local .npmrc and make it more difficult to unintentionally commit auth secrets.
+.npmrc
```

Comment:
should we "migrate" this to pnpm?

Reply:
actually can we delete the file? I'm not sure skuba would be managing it

Reply:
I confused myself, local `lint` doesn't run the subpackages. Feels weird 🤔

Reply:
I'm thinking the changes to the subpackages won't be as frequent, so we could add a lint:all, or lint:packages for them? (Just thinking out loud)